internal/alertmanager/upstream.go

```diff
@@ -78,3 +75,19 @@ func GetAlertmanagerByName(name string) *Alertmanager {
 	}
 	return nil
 }
+
+// WithProxy option can be passed to NewAlertmanager in order to enable request
+// proxying for unsee clients
+func WithProxy(proxied bool) Option {
+	return func(am *Alertmanager) {
+		am.ProxyRequests = proxied
```

Comment:
you removed `ProxyRequests` from the struct

Reply:
It's still part of the struct, I don't set it to any value by default, so (since it's a `bool`) will default to Go default, which is false, which is desirable